packages/design-system-react/src/components/icon/Icon.test.tsx

```diff
@@ -70,6 +70,71 @@ describe('Icon', () => {
     const icon = container.firstChild;
     expect(icon).toHaveStyle(customStyle);
   });
+
+  describe('className overrides', () => {
```

Comment:
Adding a check to ensure tailwind classnames for color and size are overriding the IconColor and IconSize enums